generic/account_supplier_billing/models/supplier_billing.py

```diff
@@ -20,30 +20,46 @@ class SupplierBilling(models.Model):
         'res.partner',
         string='Supplier',
         domain=[('supplier', '=', True), ],
-        states={'billed': [('readonly', True)]},
+        states={
```

Comment:
It works, but to be better,
```
        readonly=True,
        states={'draft': [('readonly', False)]},
```
Reason: Code can be shorter, more specific. In future, if we have more states, we don't need to change code.